website/docs/language/functions/toset.mdx

```diff
@@ -9,7 +9,9 @@ description: The toset function converts a value to a set.
 
 Explicit type conversions are rarely necessary in Terraform because it will
 convert types automatically where required. Use the explicit type conversion
```

Comment:
```suggestion
Explicit type conversions are rarely necessary in Terraform because it converts types automatically where required. Use the explicit type conversion
```

Suggest changing "will convert" to "converts" to avoid future tense.